src/Language/PureScript/CodeGen/JS/Optimizer/Inliner.hs

```diff
@@ -66,6 +67,8 @@ evaluateIifes = everywhereOnJS convert
   where
   convert :: JS -> JS
   convert (JSApp _ (JSFunction _ Nothing [] (JSBlock _ [JSReturn _ ret])) []) = ret
+  convert (JSApp _ (JSFunction _ Nothing idents (JSBlock _ [JSReturn ss ret])) [])
```

Comment:
Ah right, so this takes `(function(x, y, z) { return x + y + z })()` to `undefined + undefined + undefined`?